inkstitch.py

```diff
@@ -6,15 +6,30 @@
 import os
 import sys
 from pathlib import Path  # to work with paths as objects
-import configparser   # to read DEBUG.ini
 from argparse import ArgumentParser  # to parse arguments and remove --extension
 
+import configparser   # to read DEBUG.ini
+import toml           # to read logging configuration from toml file
```

Comment:
While I understand your arguments in favor of TOML, it seems suboptimal to use both configparser and toml.  Can we switch one to the other format?  DEBUG.ini is pretty new so it wouldn't be too bad to switch it to TOML since the syntax is similar...